### DISCUSSION TITLE
Invoice app

### DISCUSSION THREAD

**Discussion by lkostrowski** [saleor/apps#9] [Integrations & features]
Replacement for invoice plugin

**Comment by lkostrowski** [comment]
Done, available in the App Store https://docs.saleor.io/docs/3.x/developer/app-store/apps

**Reply by Kamal199806** [reply]
Page Not Found
We could not find what you were looking for.

Please contact the owner of the site that linked you to the original URL and let them know their link is broken.

**Reply by lkostrowski** [reply]
https://github.com/saleor/example-app-invoices
